modules/auth/views/login.php

```diff
@@ -17,39 +17,87 @@
 		}
 	?>
 
-	<hr>
+	<br />
 
+	<h1>Log in to ITRS OP5 Monitor</h1>
+	<br />
+	
 	<fieldset>
-		<label for="username">Username</label>
-		<input type="text" name="username" id="username" placeholder="Username" autofocus>
+		<label for="username">Username</label> <br />
+		<input type="text" name="username" id="username" autofocus>
 	</fieldset>
 
+	<br />
+
 	<fieldset>
-		<label for="password">Password</label>
-		<input type="password" name="password" id="password" placeholder="Password">
+		<label for="password">Password</label> <br />
+		<input type="password" name="password" id="password" >
 	</fieldset>
 
+	<br />
+
 	<?php
 
 		$auth = Auth::instance();
 		$default_auth = $auth->get_default_auth();
 		if (!empty($auth_modules) && is_array($auth_modules) && count($auth_modules) > 1) {
 			echo '<fieldset>';
 			echo form::label('auth_method', _('Login method'));
+			echo '<br />';
 			echo form::dropdown(array(
 				'name' => 'auth_method'
 			), array_combine($auth_modules, $auth_modules), $default_auth);
 			echo '</fieldset>';
-		}
+	}
 	?>
 
-	<hr>
+		<br />
 
 	<fieldset type="buttons">
-		<input type="submit" value="Login">
+		<input type="submit" value="Log in">
 	</fieldset>
 
+	<p class="rights">&#169;2019 ITRS GROUP LTD, ALL RIGHTS RESERVED</p>
+
 	<?php
 		echo form::close()
 	?>
+
+	<script>
+		$(document).ready(function(){
```

Comment:
Not sure how this works.. But is it possible to put the styling into a css file?